sdk/src/message.rs

```diff
@@ -165,9 +165,17 @@ pub struct Message {
 
 impl Sanitize for Message {
     fn sanitize(&self) -> std::result::Result<(), SanitizeError> {
-        if self.header.num_required_signatures as usize > self.account_keys.len() {
+        if self.header.num_required_signatures as usize
+            + self.header.num_readonly_unsigned_accounts as usize
+            > self.account_keys.len()
+        {
+            return Err(SanitizeError::IndexOutOfBounds);
+        }
+
+        if self.header.num_readonly_signed_accounts > self.header.num_required_signatures {
             return Err(SanitizeError::IndexOutOfBounds);
         }
+
```

Comment:
Same for check below: should be `>=` to ensure at least one RW account

Reply:
I think this check can be removed actually.

Since `num_readonly_signed_accounts < num_required_signatures`
and
 `(num_required_signatures + num_readonly_unsigned_accounts) < keys.len()` 

then this has to be true: 
`(num_readonly_signed_accounts + num_readonly_unsigned_accounts) < keys.len()`

Reply:
agree?

Reply:
Yep! I agree 🔪